src/js/injection_script.js

```diff
@@ -2711,6 +2711,16 @@ domainBypass("acorta-link.com", () => {
     })
 })
 
+domainBypass("clk.asia", () => {
+    ensureDomLoaded(() => {
+      const token = document.querySelector('#link-view [name="token"]').value;
+      const decoded = atob(token.substring(token.indexOf("aH")));
```

Comment:
dont we also want aHR0 here?

Reply:
You're absolutely right. I updated and committed. That's the best "solution" till we find a better way to identify the beginning of a base64 string